test/request/reducers/RequestDetailRequestReducers.spec.ts

```diff
@@ -0,0 +1,190 @@
+import { IMessage, IMessageEnvelope } from '../../../src/request/messages/IMessageEnvelope';
+import { IMiddlewareStartPayload } from '../../../src/request/messages/IMiddlewareStartPayload';
+import { IMiddlewareEndPayload } from '../../../src/request/messages/IMiddlewareEndPayload';
```

Comment:
In general, the only models coming from the server are messages and I think we have a fairly straightforward naming convention there (e.g. `IMessage`, `IMessageEnvelope`, `IXxxPayload`, etc.).  I've tried to name "view models" consistently as well (e.g. `IWebServicesProps`, `IWebServicesDispatchProps`, etc.) and in a way that maps well to their containers.

There is the transfer of data between selector and container that could be explicitly described in interfaces, but I've found that it generally doesn't have to as TypeScript does a pretty good job dealing with the implied types.